modules/auxiliary/scanner/mssql/mssql_ping.rb

```diff
@@ -16,6 +16,10 @@ def initialize
       'License'        => MSF_LICENSE
     )
 
+    register_options(
+      [
+        OptInt.new('THREADS', [true, "The number of concurrent threads (max one per host)", 1]),
+      ])
```

Comment:
Is this needed? I thought it would come from the scanner mixin by default 👀 

Or is it registered as an advanced option or similar?

Reply:
Same as https://github.com/rapid7/metasploit-framework/pull/18696#discussion_r1471970524